dandi/publish/views/stats.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib.auth import get_user_model
+from django.db.models import Sum
+from django.http import JsonResponse
+from rest_framework.viewsets import ViewSet
+
+from dandi.publish.models import Asset, DraftVersion, Version
+
+
+class StatsViewSet(ViewSet):
+    def list(self, request):
+        draft_count = DraftVersion.objects.count()
+        published_count = Version.objects.values('dandiset').distinct().count()
```

Comment:
Is this the correct logic? This seems to be essentially "the number of dandisets which are published", rather than "the number of versions which are published".

Reply:
This isn't implemented in girder so the only specification is the name "published_count". My thinking is that end users who see `Published Count: 100` would not expect 1 dandiset published 100 times, they would expect 100 dandisets that have been published.

Reply:
That makes sense. Perhaps to make it conceptually simpler for us to maintain, we could tack 2 classmethods onto the `Dandiset` class: `count_drafts` and `count_published`, since Dandisets are what we're ultimately counting?